src/main/src/main/java/org/geoserver/catalog/impl/DimensionInfoImpl.java

```diff
@@ -39,6 +40,10 @@
 
     DimensionDefaultValueSetting defaultValue;
     
+    Boolean nearestMatchEnabled;
```

Comment:
This a is a ``Boolean`` because:
1. we want to be backwards compatible.
2. this value may be set by reflection ?

I'm in doubt since the setter accepts a ``boolean``.

Reply:
XStream sets the value by reflection skipping the setters, yes